core/src/test/resources/META-INF/services/org.apache.spark.scheduler.ExternalClusterManager

```diff
@@ -1 +1,2 @@
-org.apache.spark.scheduler.DummyExternalClusterManager
\ No newline at end of file
+org.apache.spark.scheduler.DummyExternalClusterManager
+org.apache.spark.scheduler.MockExternalClusterManager
```

Comment:
I did not, mostly because I thought Dummy should be left as is, to keep ExternalClusterManagerSuite cleanly focused.  It would be easy to combine them, though.  Dummy isn't covering any cases that Mock isn't -- I could remove Dummy, and make ExternalClusterManagerSuite reference Mock, without making any changes to Mock.